### ISSUE TITLE
Demo voice

### ISSUE THREAD

**Issue by alterechtschreibung** [MycroftAI/mimic#126]
If you compile the sources as described in your manual, the resulting voice  not as good as the demo voice here: https://www.youtube.com/watch?v=gvnhcNdXJsk

How to get such a brilliant voice?

**Comment by penrods** [comment]
The videos from that time period was created using another online Text to Speech tool.  You'll notice the text is all forward speaking "we are going to..." type statements.  We will be continuing work  on creating more natural sounding TTS tools, but in the short term the local engine running on hardware like a Pi will not be able to produce sound like that in real-time.  For the time being, that will need to come from a cloud-based voice service.

**Comment by LongBoolean** [comment]
I had a conversation with RyanLeeSipes a while ago, about the same time
that video was released. He told me that the voice in this particular
promotional video was a non synthesized recording of Alan Pope (the person
whose voice was sampled and used to create the mimic voice). Yeah, what the
promotional video portrays and what we actually have don't match. That is
advertising for you. Personally I would have made it more clear that this
was simulated enactment of the technology and not a demo of the actual
technology itself. I wasn't going to say anything since I am not on the
Mycroft team, but I do think Spenrod's reply is inaccurate and may cause
confusion. (no offense intended, PR is hard)

@Spenrod
I think your comment is referring to the old voice like shown in this
video?: https://www.youtube.com/watch?v=-c8kfupIbO4
I am a bit confused by your comment because this voice is not actually used
in the "Introducing Mimic" promotional video in question. Also, no offense
to that other TTS, but the old voice doesn't sound very "brilliant" to me,
nor do I see how the old voice can be confused with being identical to the
voice actor recording. I am not going to assume any nefarious motive. What
likely happened is you looked at the date of the video, saw that it was an
old video made during the time when the old voice was used, and then
repeated a similar PR message, because I am sure you get the question, "why
does the voice sound different in this video", a lot.

TLDR: That brilliant voice is not a demo, PR is hard.

**Comment by Motherboard** [comment]
What about the demo from mimic II (April 2018)?

**Comment by zeehio** [comment]
I don't know, but maybe they used: https://github.com/MycroftAI/mimic2 ?
